edxval/api.py

```diff
@@ -1051,7 +1051,7 @@ def import_from_xml(xml, edx_video_id, resource_fs, static_dir, external_transcr
 
         return edx_video_id
     except ValidationError as err:
-        logger.exception(err.message)
+        logger.exception(xml)
```

Comment:
Is the error message not useful? Can we provide some thing other than just the raw XML?

Reply:
It's redundant. `logger.exception()` will already include the exception message in the stacktrace. The argument is just extra info to include.